live-examples/css-examples/css/padding.css

```diff
@@ -0,0 +1,6 @@
+#example-element {
+    border: solid 10px #FFC129;
+    font: Georgia, sans-serif;
```

Comment:
OCD: Not really using the `font` shorthand here so, should be specified as `font-family`